spec/controllers/course/mail_templates_controller_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+RSpec.describe Course::MailTemplatesController, :type => :controller do
+  before do
+    ActsAsTenant.current_tenant = Instance.default
+    sign_in(create(:administrator))
+  end
+  let!(:course) { create(:course) }
+  let(:existing) { create(:invitation, course: course) }
+  let(:new_template) { build(:invitation, course: course).as_json }
+
+  def expect_correct_response(redirect_path, controller_action)
+    expect(response).to redirect_to(redirect_path)
+    expect(flash[:notice]).
+      to match(I18n.t("course.mail_templates.#{controller_action}.notice_format") %
+                 { action: I18n.t("course.mail_templates.action.invitation") })
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.